l10n_ar_ux/models/account_chart_template.py

```diff
@@ -9,9 +9,8 @@ class AccountChartTemplate(models.AbstractModel):
 
     _inherit = 'account.chart.template'
 
-    def _load(self, template_code, company, install_demo):
-        """ Set non monetary tag when installing chart of account """
-        res = super()._load(template_code, company, install_demo)
-        if template_code in ('ar_base', 'ar_ex', 'ar_ri'):
-            self.env['account.account'].set_non_monetary_tag(company)
+    def _load_data(self, data, ignore_duplicates=False):
+        res = super()._load_data(data)
+        if res.get('res.company'):
```

Comment:
we are sure that the return is always a dictionary?

Reply:
This is the method called on "super" https://github.com/odoo/odoo/blob/18.0/addons/account/models/chart_template.py#L498 and this is the return https://github.com/odoo/odoo/blob/18.0/addons/account/models/chart_template.py#L560 . If I am not wrong, apparently it always returns a dictionary.